exchange_server/check.py

```diff
@@ -0,0 +1,110 @@
+# (C) Datadog, Inc. 2013-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# datadog
+try:
+    from checks.libs.win.pdhbasecheck import PDHBaseCheck
+except ImportError:
+    def PDHBaseCheck(*args, **kwargs):
+        return
+
+DEFAULT_COUNTERS = [
```

Comment:
Nit, can you unify the spacing here? In some you have a space between the class of metrics and in others you have no space. (I think all of them having a space might make the most sense)